marshal.go

```diff
@@ -1726,6 +1726,19 @@ func unmarshalUUID(info TypeInfo, data []byte, value interface{}) error {
 		return nil
 	}
 
+	if len(data) != 16 {
+		return unmarshalErrorf("Unable to parse UUID: UUIDs must be exactly 16 bytes long")
```

Comment:
**nitpick**: Error messages in Go should start with lowercase letters, please change `Unable` to `unable`.

https://github.com/golang/go/wiki/CodeReviewComments#error-strings

(I know there is error starting with uppercase letter below, I have opened a separate pull request to make the other errors lowercase: #1506)

Reply:
Done!

Reply:
Thanks!